mlflow/models/model.py

```diff
@@ -54,7 +53,7 @@ def __init__(
         self.flavors = flavors if flavors is not None else {}
         self.signature = signature
         self.saved_input_example_info = saved_input_example_info
-        self.model_uuid = uuid.uuid4().hex if model_uuid is None else model_uuid
+        self.model_uuid = uuid.uuid4().hex
```

Comment:
This basically means every `Model` instance has a different `model_uuid`. Is this really desired behavior?

Reply:
Every new constructed model has a different ID, this is desired.
We only need keep the rule that the model load back from saved model should load old ID back or set it None if saved model don't have ID